draft-ietf-masque-connect-ip.md

```diff
@@ -430,6 +430,26 @@ decremented right before an IP packet is transmitted in an HTTP Datagram. This
 prevents infinite loops in the presence of routing loops, and matches the
 choices in IPsec {{?IPSEC=RFC4301}}.
 
+IPv6 requires that every link have an MTU of at least 1280 bytes
+{{!IPv6=RFC8200}}. Since CONNECT-IP conveys IP packets in HTTP Datagrams and
+those can in turn be sent in QUIC DATAGRAM frames which cannot be fragmented
+{{!DGRAM=RFC8221}}, the MTU of a CONNECT-IP link can be limited by the MTU of
+the QUIC connection that CONNECT-IP is operating over. This can lead to
+situations where the IPv6 minimum link MTU is violated. CONNECT-IP endpoints
+that support IPv6 MUST ensure that the link MTU is at least 1280. This can be
+accomplished using various techniques:
+
+* if HTTP intermediaries are not in use, CONNECT-IP endpoints can pad QUIC
+  INITIAL packets.
+
+* if HTTP intermediaries are in use, CONNECT-IP endpoints can enter in an out of
+  band agreement with the intermediaries to ensure that endpoints and
+  intermediaries pad QUIC INITIAL packets.
```

Comment:
I think the main problem with this is that a Connect-IP client can't know if there are HTTP intermediaries in place or not when sending a request to the server. Only the server may actually know this, and depending on infrastructure it might be hard for an application to be certain of this even. So is this a strict requirement on the server?